modules/swagger-codegen/src/main/resources/Java/build.gradle.mustache

```diff
@@ -11,19 +11,36 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:1.5.+'
         classpath 'com.github.dcendents:android-maven-gradle-plugin:1.3'
+        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.3'
     }
 }
 
+apply plugin: 'com.jfrog.bintray'
+
 repositories {
     jcenter()
 }
 
+bintray {
+    user = System.getenv('BINTRAY_USER')
+    key = System.getenv('BINTRAY_KEY')
+
+    configurations = ['archives'] //When uploading configuration files
+    publish = true
+    pkg {
+        repo = 'maven'
+        name = 'smooch-api'
```

Comment:
We talked about this offline but in addition to the fact that this name is `smooch-api`, it doesn't actually mean that that is what people will be importing. It's just the name of the package in bintray. This name is almost meaningless.